src/containers/import-modal.jsx

```diff
@@ -50,8 +50,7 @@ class ImportModal extends React.Component {
         this.setState({inputValue: e.target.value, hasValidationError: false});
     }
     validate (input) {
-        const urlMatches = input.match(
-            /^(https:\/\/)?scratch\.mit\.edu\/projects\/(\d+)(\/?|(\/#((editor)|(fullscreen)|(player)))?)$/);
+        const urlMatches = input.match(/^(https:\/\/)?scratch\.mit\.edu\/projects\/(\d+)/);
```

Comment:
One note here: this isn't too important, but you can use a non-capturing group for the https check with `(?:https:\/\/)` which will clean up the `urlMatches` array a little